src/main/java/annotator/model/pack/InvalidFileFormatException.java

```diff
@@ -0,0 +1,19 @@
+package annotator.model.pack;
+
+public class InvalidFileFormatException extends Exception {
```

Comment:
There is FileFormatException. It would be better to use it and not create new one.